spec/models/embedded_ansible_worker/runner_spec.rb

```diff
@@ -13,26 +13,35 @@
       described_class.new(:guid => worker_guid)
     }
 
-    context "#update_embedded_ansible_manager" do
+    context "#update_embedded_ansible_provider" do
+      before do
+        EvmSpecHelper.local_guid_miq_server_zone
+        MiqDatabase.seed
+        MiqDatabase.first.ansible_admin_password = "secret"
```

Comment:
When did this get added?  Did I miss a migration?  I know we discussed it but I don't remember the PR.

Even so, since we've added it, is it encrypted / fix_authable?